### PR TITLE
Document user session endpoints for APIv4

### PR DESCRIPTION
**Pull request by ruzette**

### CODE REVIEW

File: source/v4/users.yaml [hunk 1]
```diff
@@ -485,3 +485,70 @@
           $ref: '#/responses/Unauthorized'
         '404':
           $ref: '#/responses/NotFound'
+
+  /users/{user_id}/sessions:
+    get:
+      tags:
+        - users
+      summary: Get a list of sessions by User GUID.
```
Comment (jwilander) [inline source/v4/users.yaml]: Can we simplify the summary to just be `Get a user's sessions`? It'll be showing up in a sidebar so it's better if we can keep the summaries short :)

File: source/v4/users.yaml [hunk 2]
```diff
@@ -485,3 +485,70 @@
...
+        '401':
+          $ref: '#/responses/Unauthorized'
+        '403':
+          $ref: '#/responses/Forbidden'
+ 
+  /users/{user_id}/sessions/revoke:
+    post:
+      tags:
+        - users
+      summary: Get a list of sessions by User GUID.
```
Comment (jwilander) [inline source/v4/users.yaml]: I think this should be `Revoke a session` right?

File: source/v4/users.yaml [hunk 3]
```diff
@@ -485,3 +485,70 @@
...
+      description: |
+        Get a list of sessions by providing the user GUID. Sensitive information will be sanitized out.
```
Comment (jwilander) [inline source/v4/users.yaml]: I think you forgot to update this too

File: source/v4/users.yaml [hunk 4]
```diff
@@ -485,3 +485,70 @@
...
+            type: object
+            required:
+              - session_id
+            properties:
+              session_id:
+                description: The session GUID to be revoked.
+                type: string
+      responses:
+        '200':
+          description: User session retrieval successful
```
Comment (jwilander) [inline source/v4/users.yaml]: Here as well

**Comment by ruzette** [comment]
Oops. apologies about that. I thought I pushed that commit prior to the PR. @jwilander ready for review!

**Pull request merged**